src/rev_ai/streamingclient.py

```diff
@@ -81,6 +85,9 @@ def start(self, generator, metadata=None, custom_vocabulary_id=None):
         if metadata:
             url += '&' + urlencode({'metadata': metadata})
 
+        if filter_profanity:
+            url += '&' + urlencode({'filter_profanity': 'true'})
```

Comment:
personal preference, but thoughts on using a format string here?

Reply:
Also should we be defaulting this option to true?

Reply:
This is not defaulting to true. The default is None ([this line](https://github.com/revdotcom/revai-python-sdk/pull/45/files/29d2d44347d33919b3a0cfae7a1694a6a04f0a9d#diff-c79255447727fc4d20e17512f19774aeR69)).

As for format strings, if I recall correctly, usage of the library is more desirable as it handles any url formatting issues that might not be caught otherwise.